File: 1.C and C++/1.Array/1.Finding a element array.cpp
#include <iostream>

using namespace std;

int findElement(int arr[], int x, int n)
{
    for(int i=0; i<n; i++)
    {
        if(arr[i] == x)
            return i;
    }

    return -1;
}
int main()
{
    int arr[50] = {20, 35, 44, 7, 52};
    int x = 44;

    cout<<findElement(arr,x,5);
    return 0;
}

File: 1.Java/1.Array/1.Finding a element array.java
import java.io.*;
import java.util.*;

public class Test {

     public static int findElement(int arr[], int x, int n)
     {
         for(int i=0; i<n; i++)
             if(arr[i] == x)
                 return i;

         return -1;
     }

     public static void main(String args[])
     {
         int arr[] = {33,5,87,3,24};
         int x = 8;

         System.out.println(findElement(arr,x,5));
     }
}
